tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-chart.html

```diff
@@ -56,59 +52,15 @@
       </template>
     </div>
     <div style="display: flex; flex-direction: row;">
-      <paper-icon-button
-        selected$="[[_expanded]]"
-        icon="fullscreen"
-        on-tap="_toggleExpanded"
-      ></paper-icon-button>
-      <paper-icon-button
-        selected$="[[_logScaleActive]]"
-        icon="line-weight"
-        on-tap="_toggleLogScale"
-        title="Toggle y-axis log scale"
-      ></paper-icon-button>
-      <paper-icon-button
-        icon="settings-overscan"
-        on-tap="_resetDomain"
-        title="Fit domain to data"
-      ></paper-icon-button>
-      <span style="flex-grow: 1"></span>
-      <template is="dom-if" if="[[showDownloadLinks]]">
-        <div class="download-links">
-          <paper-dropdown-menu
-            no-label-float="true"
-            label="run to download"
-            selected-item-label="{{_runToDownload}}"
-          >
-            <paper-menu class="dropdown-content" slot="dropdown-content">
-              <template is="dom-repeat" items="[[runs]]">
-                <paper-item no-label-float=true>[[item]]</paper-item>
-              </template>
-            </paper-menu>
-          </paper-dropdown-menu>
-          <a
-            download="run_[[_runToDownload]]-tag-[[tag]].csv"
-            href="[[_csvUrl(_runToDownload)]]"
-          >CSV</a> <a
-            download="run_[[_runToDownload]]-tag-[[tag]].json"
-            href="[[_jsonUrl(_runToDownload)]]"
-          >JSON</a>
-          </div>
-        </div>
-      </template>
+      <content select=".buttons"></content>
```

Comment:
I'd recommend against having `tf-scalar-chart` own the DOM for the buttons: we had something like this before I removed all the chart scaffolding logic, and anything nontrivial required a super hacky implementation, was not accessible, etc.

Let's just have the parent element write
```html
<tf-scalar-chart … />
<div>
  <!-- buttons -->
</div>
```
If there's a benefit to doing it as currently written that I'm not seeing—maybe in terms of CSS/layout?—I'm sure that there is an easy way to redesign it for equivalent functionality.

Reply:
Done. Indeed, I see no major benefits there - maybe we write the CSS for buttons once, but that's not too compelling.